src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
The property 'spring.application.name' is missing. It is important to define the application name in the properties file.